.clang-format

```diff
@@ -1,4 +1,5 @@
 ﻿---
 BasedOnStyle: WebKit
 ColumnLimit: 88
+SortIncludes: false
```

Comment:
I don't like adding this.

Reply:
Yeah, me neither. Letting `clang-format` sort includes incurs into compilation issues. Your fix solve them partially, but there are more other issues to solve. I think solving these issues manually is not an option (we don't want to do this everytime we clang-format the code base). So I think we have two options:
1. Reject the PR until we can correctly solve the include dependencies;
2. Accept PR and push the solution of include dependencies for later;

Reply:
I vote for 1 purely so we don't end up putting off the merge indefinitely again...

Reply:
I vote for 2. 

But I don't understand how `clang-format` is to blame? If our `include` order is causing compilation issues then we must have some messed up dependencies in our files and that should be fixed.  

I will raise an issue for this.